documentation/modules/upgrading-mtv-ui.adoc

```diff
@@ -55,7 +50,7 @@ ifeval::["{build}" == "downstream"]
 See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/operators/administrator-tasks#olm-approving-pending-upgrade_olm-upgrading-operators[Manually approving a pending upgrade] in the {ocp} documentation.
 endif::[]
 
-. Verify that the `forklift-ui` pod is in a `Ready` state before you log in to the web console:
+. Verify that the {project-short} pods are running:
```

Comment:
thinking about this again, previously it was necessary so one won't log-in the the OCP console when the route to the old UI doesn't exist - but now with the console-plugin it's no longer needed since the user gets a notification that the console has changed and needs to be refreshed, right? I think we can drop this part of verifying that the pods are running

Reply:
@yaacov ^

Reply:
OK, step, command and output removed.